Automattic/jetpack
# [Plugin] Backup: update package usage and add linting.

#### Changes proposed in this Pull Request:

- Linting: add linting for i18n in the admin interface.
- Simplify use of `@wordpress/element`

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Functionality should not change.
* Go to the Backup menu and check that different items are properly displayed.
* Edit a translatable string to use no textdomain or a wrong textdomain, and see that `pnpm lint-file filepath` returns an error.
